### PR TITLE
More pages

### PR DESCRIPTION
**Pull request by jasonbriggs**
-much better blending of template styles on non lander pages
-new tagline and explainer text
-responsive fixes on lander
-new pricing page
-new about us page
-new guarantee page
-other UI tweaks and improvements

### CODE REVIEW

File: layouts/section/pricing.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                <div class="cd-pricing-body">
+                    <ul class="cd-pricing-features">
+                        <li><em>Two</em> Cloud Integrations</li>
+                        <li class="right"><em>One</em> Synced Device</li>
+
+                    </ul>
+                </div> <!-- .cd-pricing-body -->
+
+                <footer class="cd-pricing-footer">
+                    <a id="basic" class="cd-select" href="/">Sign Up</a>
```
Comment (kujenga) [inline layouts/section/pricing.html]: I think having these just link back to the homepage makes it seem like the site's broken. Perhaps they should just be disabled for now? We could add "Coming Soon" or similar to explain why they're disabled

File: layouts/section/about.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    <!-- Navigation -->
+    {{ partial "nav.html" "metafeatures" }}
+    <section class="box special features" style="border:none;box-shadow:none;outline:none;">
+        <div id="headerspace"></div>
+        <section id="aboutus">
+            <h1> About Us</h1>
+            <p class="jobtext">
+                <b>It all began with a Desktop that looked like this. </b>Jason, a cofounder of Meta, was helping his mom track down some of her files (and failing). She had 356 files on her Desktop alone, and was constantly creating and receiving more. There was no way to bring order to the chaos.
+            </p>
+            <img id="desktop" src="/assets/images/desktop_capitan.png">
```
Comment (kujenga) [inline layouts/section/about.html]: It doesn't look like this image has been committed, I'm getting a blank placeholder here

File: layouts/partials/nav.html [hunk 3]
```diff
@@ -20,14 +20,18 @@
...
                     <li>
-                        <a class="page-scroll" href="/tech">Tech</a>
+                        <a class="page-scroll" href="/blog/">Blog</a>
                     </li>
                     <li>
-                        <a class="page-scroll" href="/careers">Careers</a>
+                        <a class="page-scroll" href="/tech/">Tech</a>
                     </li>
+                    <li>
+                        <a class="page-scroll" href="/about/">About</a>
```
Comment (kujenga) [inline layouts/partials/nav.html]: I think we should reorder this to be next to pricing so that it matches the homepage

Comment (jasonbriggs) [inline layouts/partials/nav.html]: Good call!

File: layouts/section/about.html (hunk 2 above)
Comment (jasonbriggs) [inline layouts/section/about.html]: good catch!

File: layouts/section/pricing.html [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                    <a id="basic" class="cd-select">Coming Soon</a>
+                </footer> <!-- .cd-pricing-footer -->
+            </li>
+            <li class="cd-popular">
+                <header class="cd-pricing-header">
+                    <h2>Pro</h2>
+
+                    <div class="cd-price">
+                        <span class="cd-currency">$</span>
+                        <span class="cd-value">5</span>
```
Comment (kujenga) [inline layouts/section/pricing.html]: Lets bump these up to the higher end of the values we're considering, so 10 here and 20 for enterprise. Always better for prices to come down than go up!

**Comment by kujenga** [comment]
Looks great! Nice work

**Pull request merged**